mgnify_pipelines_toolkit/analysis/shared/study_summary_generator.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python
```

Comment:
I think you branched from the wrong branch. We should merge the study summary changes before this one so that this disappears from this PR.

Reply:
ah.. well. I think we can just merge it as it is